stages/org.osbuild.qemu

```diff
@@ -24,13 +24,19 @@ SCHEMA_2 = r"""
     "description": "Create a qcow2 image",
     "type": "object",
     "required": ["type"],
+    "not": {"required": ["disable-compression", "compat"]},
     "additionalProperties": false,
     "properties": {
       "type": {
         "description": "The type of the format, here 'qcow2'",
         "type": "string",
         "enum": ["qcow2"]
       },
+      "disable-compression": {
```

Comment:
I almost feel like it would be cleaner to name this `compression` and have it default to true.

Reply:
+1